_mobile/android.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: mobile
+title: Android
+permalink: /mobile/android/
+background-class: mobile-background
+body-class: mobile
+order: 3
+published: true
+---
+
+# Android
+
+## Quickstart with a HelloWorld Example
+
+[HelloWorld](https://github.com/pytorch/android-demo-app/tree/master/HelloWorldApp) is a simple image classification application that demonstrates how to use PyTorch Android API.
+This application runs TorchScript serialized TorchVision pretrained resnet18 model on static image which is packaged inside the app as android asset.
+
+#### 1. Model Preparation
+
+Let’s start with model preparation. If you are familiar with PyTorch, you probably should already know how to train and save your model. In case you don’t, we are going to use a pre-trained image classification model(Resnet18), which is packaged in [TorchVision](https://pytorch.org/docs/stable/torchvision/index.html).
```

Comment:
looks like link got lost - copy from the iOS tutorial:

`[Resnet18](https://pytorch.org/hub/pytorch_vision_resnet/)`